homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import Form from './Components/Form';
+import List from './Components/List';
+import './App.css';
+
+@inject('TasksStore')
+@observer
+
+class App extends Component {
+  render() {
+    const { TasksStore } = this.props;
+    return (
+      <div className="App">
+        <Form
+          onAdd={TasksStore.handleAdd}
+
+        />
+        <List
+          handleCheck={TasksStore.handleCheck}
```

Comment:
As I said in the previous comment, None of these props are in use inside of the `List` Component.